client/js/render.js

```diff
@@ -77,12 +73,6 @@ function appendMessage(container, chanId, chanType, msg) {
 
 function buildChatMessage(chanId, msg) {
```

Comment:
ChanId should be removed... which will create a fun diff. Feel free to leave it out of this PR for that reason.

Reply:
It's called in a single place, so it's easy.